hubblestack/extmods/modules/win_pulsar_winaudit.py

```diff
@@ -552,3 +568,39 @@ def _dict_update(dest, upd, recursive_update=True, merge_lists=False):
             for k in upd:
                 dest[k] = upd[k]
         return dest
+
+def top(topfile='salt://hubblestack_pulsar/win_top.pulsar',
```

Comment:
lost in commit https://github.com/hubblestack/hubble/commit/7cccf7b9e92dfbc32c0253bc3d63418962852dcd